docs/contributing/code-of-conduct.md

```diff
@@ -1,76 +1,51 @@
 
-## Scope
+## Toepasbaarheid
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+Deze Gedragscode is van toepassing zowel binnen projectruimten als in openbare ruimtes wanneer een persoon het project of de gemeenschap vertegenwoordigt.
 
-## Enforcement
+Voorbeelden van het vertegenwoordigen een project of gemeenschap zijn het gebruiken van een officieel project e-mailadres, het plaatsen via een officieel social media-account of optreden als een aangewezen vertegenwoordiger bij een online of offline evenement. Vertegenwoordiging van een project kan nader gedefinieerd en verduidelijkt zijn door projectbeheerders.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at
-[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). All complaints will be
-reviewed and investigated and will result in a response that is deemed necessary
-and appropriate to the circumstances. The project team is obligated to maintain
-confidentiality with regard to the reporter of an incident. Further details of
-specific enforcement policies may be posted separately.
+## Handhaving
 
-Project maintainers who do not follow or enforce the Code of Conduct in good
-faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
+Gevallen van beledigend, intimiderend of anderszins onaanvaardbaar gedrag kunnen worden gemeld door contact op te nemen met het projectteam op [Conduct@gatsbyjs.com](mailto: conduct@gatsbyjs.com). Alle klachten worden beoordeeld en onderzocht en zullen resulteren in een reactie die noodzakelijk wordt geacht en past bij de mstandigheden. Het projectteam is verplicht vertrouwelijk te handelen ten aanzien van tot de melder van een incident. Verdere details van specifiek handhavingsbeleid kunnen afzonderlijk worden gepubliceerd.
+
+Projectonderhouders die de Gedragscode niet goed naleven of handhaven
+kunnen worden geconfronteerd met tijdelijke of permanente repercussies zoals bepaald door andere leden van het projectleiderschap.
 
 ## Attribution
 
-This Code of Conduct is adapted from the Contributor Covenant, version 1.4.
+Deze Gedragscode is overgenomen uit het Bijdragerconvenant, versie 1.4.
 
-- homepage: https://contributor-covenant.org
-- version: https://contributor-covenant.org/version/1/4/
+- startpagina: https://contributor-covenant.org
+- versie: https://contributor-covenant.org/version/1/4/
```

Comment:
de echte vertaling ziet er anders uit, is dat verwacht?

zie: https://www.contributor-covenant.org/nl/version/1/4/code-of-conduct

Reply:
Makes sense to point to the Dutch translation as well. Updated the link. Or did you mean something else?

Reply:
I meant that the text of this document isn't the same as the one on the link

Reply:
😅 Ah, sure!